app/helpers/ember_cli_rails_assets_helper.rb

```diff
@@ -0,0 +1,23 @@
+require "ember_cli/assets/lookup"
+
+module EmberCliRailsAssetsHelper
+  def include_ember_script_tags(name, **options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.